### PR TITLE
Hardtabs "fix"

### PR DESCRIPTION
**Pull request by kbrose**
* Just return w/o indenting if using hardtabs
* Add minimal help section to readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,3 +76,12 @@ or_like_this = [
 
 There are plenty of other examples (ordinary and extraordinary) in the
 [test_file](https://github.com/DSpeckhals/python-indent/blob/master/spec/test_file.py).
+
+### This isn't working for me!
+
+Sorry to hear that! Hopefully one of the following will help you fix the problem.
+
+* Check is that you are using "soft tabs" (i.e. spaces). Hard tabs (i.e. `'\t'`) are incompatible with PEP8 compliant indenting so this package gives up if the editor is using hard tabs.
```
Comment (DSpeckhals) [inline README.md]: Technically, I think tabs are acceptable, just not for "continued indentation." I would re-word it surrounding that point.

File: README.md [hunk 2]
```diff
@@ -76,3 +76,12 @@ or_like_this = [
...
+    * Atom has two different settings for this, the "TabType" setting (with choices "auto", "soft", and "hard"), and the "Soft Tabs" setting which is used when TabType is auto, but the auto detection fails.
+* Make sure your python code is correctly written -- sometimes what looks like incorrect indentation is actually being caused by a bracket twenty lines ago that somehow lost its closing partner!
```
Comment (DSpeckhals) [inline README.md]: Capitalize "python"

**Review by DSpeckhals — CHANGES_REQUESTED** [review]
Thank you so much for doing this. I think the additions to the README too. There's just a couple of minor nitpicks I have.

File: README.md (hunk 1 above)
Comment (kbrose) [reply README.md]: note to self: also fix grammar "Check is that..."

**Comment by kbrose** [comment]
@DSpeckhals thanks for the review. Latest commit should address comments.

**Comment by kbrose** [comment]
This will close #42

**Review by DSpeckhals — APPROVED** [review]

**Pull request merged**